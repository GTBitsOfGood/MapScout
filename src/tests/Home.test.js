import React from 'react';
import Home from '../components/Index';
import renderer from 'react-test-renderer';

it('renders', () => {
  const tree = renderer
    .create(<Home/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
