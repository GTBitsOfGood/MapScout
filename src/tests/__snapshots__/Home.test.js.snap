// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div
  className="home"
>
  Home
</div>
`;
